src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,135 @@
+package duke.ui;
+
+import duke.data.messages.Messages;
+
+/**
+ * Class that deals with interactions with the user of Duke.
+ *
+ * @author Won Ye Ji
+ */
+public class Ui {
+
+    /**
+     * Greets the user.
+     */
+    public static void greet() {
+        System.out.println(Messages.WELCOME_MESSAGE);
+    }
+
+    /**
+     * Farewell message to the user.
+     */
+    public static void bye() {
+        System.out.println(Messages.INDENTATION + Messages.BYE_MESSAGE);
+    }
+
+    /**
+     * Informs the user that they have inputted an empty task.
+     * 
+     * @param s Type of task input.
+     * @return The error message to the user.
+     */
+    public static String emptyDescription(String s) {
```

Comment:
Perhaps a better name for this parameter `s` can be `taskType`?
```suggestion
    public static String emptyDescription(String taskType) {
```